src/index.html

```diff
@@ -4,14 +4,153 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="group">
+        <legend>Personal information</legend>
+        <div class="field">
+          <label>Surname:
+            <input
+              type="text"
+              name="surName"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="field">
+          <label>Name:
+            <input
+              type="text"
+              name="name"
+              required
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="field">
+          <label>How old are you?
```

Comment:
```suggestion
          <label>
            How old are you?
```
write text from new line , check it everywhere